src/util/i18n.js

```diff
@@ -0,0 +1,55 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import Backend from 'i18next-http-backend';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import { DateTime } from 'luxon';
+import translationEN from '../lang/translationEN.json';
+import translationRU from '../lang/translationRU.json';
+// don't want to use this?
```

Comment:
should probably remove these 3 lines of "quick start" comments

Reply:
removed